packages/main/src/Button.hbs

```diff
@@ -48,7 +49,7 @@
 	{{/if}}
 
 	{{#if hasButtonType}}
-		<span class="ui5-hidden-text">{{buttonTypeText}}</span>
+		<span id="{{_id}}-buttonTypeText" aria-hidden="true" class="ui5-hidden-text">{{buttonTypeText}}</span>
```

Comment:
As I know that _id is not recommended to be used and there is no need to be a dynamic id, you can use a static id because the element is in the shadow DOM of the button and will not collide with other button spans on the page.

Reply:
done